cluster/test-deploy/aws/openshift.yaml

```diff
@@ -153,7 +153,8 @@ aws:
 # To use Azure-provided DNS, `BaseDomain` should be set to `""`
 # If using DNS records, ensure that `BaseDomain` is set to a properly configured external DNS zone.
 # Instructions for configuring delegated domains for Azure DNS can be found here: https://docs.microsoft.com/en-us/azure/dns/dns-delegate-domain-azure-dns
-baseDomain: origin-ci-int-aws.dev.rhcloud.com
+#baseDomain: origin-ci-int-aws.dev.rhcloud.com
+baseDomain: openshiftdemo.org
 
```

Comment:
@stevekuznetsov This is for local testing using rh-dev account.. will change it back or put in an if_local_dev block or something

Reply:
Let's remove the local dev bits

Reply:
yup done